src/JBOmnichainDeployer.sol

```diff
@@ -332,17 +351,17 @@ contract JBOmnichainDeployer is
     function launchRulesetsFor(
         uint256 projectId,
         JBRulesetConfig[] calldata rulesetConfigurations,
-        JBTerminalConfig[] memory terminalConfigurations,
+        JBTerminalConfig[] calldata terminalConfigurations,
         string calldata memo
     )
         external
         returns (uint256)
     {
-        rulesetConfigurations = _setup({projectId: projectId, rulesetConfigurations: rulesetConfigurations});
+        // TODO: Permissions.
 
         return CONTROLLER.launchRulesetsFor({
```

Comment:
permissions already gonna get caught here right? project owner will have to give this contract operator permissions?

Reply:
Yeah they will need to, but after that anyone will be able to call this contract as it has permission. So we need to make sure that the person calling the contract also has permissions.